src/openMVG/rig/Rig.cpp

```diff
@@ -67,17 +67,16 @@ bool Rig::initializeCalibration()
   // check that there are cameras
   assert(_vLocalizationResults.size()>0);
   
-  // check that all the cameras have the same number of localizationResults
-  const std::size_t sequenceLength = _vLocalizationResults[0].size();
+  // make all the cameras have the same number of localizationResults (equal to the shortest clip)
   const std::size_t nCams = _vLocalizationResults.size();
-  
-  for(std::size_t i = 1; i < nCams; ++i)
   {
-    assert(_vLocalizationResults[i].size() == sequenceLength 
-            && "All cameras must have the same number of images");
+    std::size_t shortestSeqLength(-1);
```

Comment:
There's no overflow, `size_t(-1)` is a well-defined way of creating a `size_t` of maximum possible value. (On 2nd complement machines, but I've yet to see one which is based on one's complement AND runs openmvg).